### PR TITLE
Instead of listing all docker tags for a repo, check if the manifest is available for that tag

### PR DESCRIPTION
**Pull request by jwilander**
Not sure why the tags request was failing (happened locally for me too) but this is a better way to do it anyways. Tested locally.

### CODE REVIEW

**Comment by jwilander** [comment]
Build failure looks unrelated

**Comment by cpanato** [comment]
we need to make the build Pass, otherwise cannot deploy

**Comment by jwilander** [comment]
Read my summary @cpanato :P it was a hacky test but it was a test

File: go.mod [hunk 1]
```diff
@@ -16,6 +16,7 @@ require (
 	github.com/gorilla/mux v1.7.3
 	github.com/heroku/docker-registry-client v0.0.0-20190909225348-afc9e1acc3d5
 	github.com/mattermost/mattermost-cloud v0.7.0
+	github.com/mattermost/mattermost-server v5.11.1+incompatible
```
Comment (hanzei) [inline go.mod]: @jwilander Why is the import of this version needed?

Comment (jwilander) [reply go.mod]: I didn't pick that version specifically. I just ran `go mod vendor` to fix the lint issues so I could get this merged.

**Pull request merged**